lib/config/httpRoutes.js

```diff
@@ -153,7 +153,10 @@ module.exports = [
   {verb: 'post', url: '/credentials/:strategy/_me/_create', controller: 'auth', action: 'createMyCredentials'},
   {verb: 'post', url: '/credentials/:strategy/_me/_validate', controller: 'auth', action: 'validateMyCredentials'},
 
+  /* DEPRECATED - will be removed in v2 */  
```

Comment:
(open to debate)

since this deprecation notice is introduced so late in the v1, I wonder if this shouldn't stay for the next major version
Usually, you deprecated something during a major, it stays in the next one, and (sometimes) removed after that

Reply:
Since there is only very few people that use the validation module I think we can remove this route.
If we setup a `compat` configuration option (cf [trello](https://trello.com/c/WAcB9MHT/74-adm-ajout-dune-partie-compat-dans-la-config-de-kuzzle-pour-pouvoir-d%C3%A9sactiver-des-fonctionnalit%C3%A9s-deprecated)) users will be able to keep deprecated route if they want but IMHO it's better to push users to embrace change